language/move-lang/functional-tests/tests/on_chain_config/change_script_allowlist.move

```diff
@@ -21,13 +21,12 @@ module FooConfig {
 
 //! new-transaction
 //! sender: libraroot
-//! args: x"0100010000000000000000000000000a550c18",
 // Step 2: Change option to CustomScript
 script {
 use 0x1::LibraVMConfig;
 
-fun main(config: &signer, args: vector<u8>) {
-    LibraVMConfig::set_publishing_option(config, args)
+fun main(config: &signer) {
+    LibraVMConfig::set_open_module(config, false)
```

Comment:
Intended? This doesn't match the comment on line 24

Reply:
That's intended, cuz the functional test first set the publishing option to be open script and we would like to then disable module publishing. Lemme update the comment